ensemble_eeg/ensemble_edf.py

```diff
@@ -219,7 +219,26 @@ def write_edf_header(fd, header):
                 if not isinstance(val, bytes):
                     val = str(val).encode(encoding="ascii").ljust(size, b"\x20")
```

Comment:
```suggestion
                    val = bytes(val, encoding="ascii").ljust(size, b" ")
```

Also change the above line:
```python
                    val = b"\x20" * size
```

Reply:
Perhaps fix this as well? Other than that, looks good to me.